### ISSUE TITLE
Merged Ticket escalated

### ISSUE THREAD

**Issue by tobibe** [zammad/zammad#904]
### Infos:

* Used Zammad version: 1.4.0
* Operating system: Ubuntu 16.04.02
* MySQL Server: 5.6
* Browser + version: FireFox 52.0.1 32Bit

Merged Ticket does not match SLA conditions, but is shown in standard escalated overview
![grafik](https://cloud.githubusercontent.com/assets/9305993/24413055/9647cf1c-13da-11e7-9946-d1f803fcb2cd.png)
Sometimes Ticket escalate, but don't fit escalation conditions any more. The Ticket is displayed correctly with the non matching parameters. If you update the ticket once more, the escalation disappear.

I found no way to reproduce it.

I copied the database entries of the ticket shown in picture:

Entry in Table ticket
id, group_id, priority_id, state_id, organization_id, number, title, owner_id, customer_id, note, first_response_at, first_response_escalation_at, first_response_in_min, first_response_diff_in_min, close_at, close_escalation_at, close_in_min, close_diff_in_min, update_escalation_at, update_in_min, update_diff_in_min, last_contact_at, last_contact_agent_at, last_contact_customer_at, create_article_type_id, create_article_sender_id, article_count, escalation_at, pending_time, type, preferences, updated_by_id, created_by_id, created_at, updated_at, internalwaitfor, time_unit
'1225', '16', '2', '5', NULL, '2201225', 'Title', '1', '197', NULL, NULL, '2017-03-23 08:28:23.000', NULL, NULL, NULL, NULL, NULL, NULL, '2017-03-23 08:28:24.000', '0', '1440', '2017-03-20 09:58:23.255', NULL, '2017-03-20 09:58:23.255', '10', '1', '1', '2017-03-23 08:28:23.000', NULL, NULL, '--- !ruby/hash:ActiveSupport::HashWithIndifferentAccess\nescalation_calculation: !ruby/hash:ActiveSupport::HashWithIndifferentAccess\n  first_response_at: \n  last_update_at: 2017-03-20 09:58:23.255000000 Z\n  close_at: \n  sla_id: 2\n  sla_updated_at: 2017-02-27 15:40:33.303000000 Z\n  calendar_id: 1\n  calendar_updated_at: 2017-03-19 13:11:10.462000000 Z\n  escalation_disabled: false\n', '8', '9', '2017-03-20 09:58:23', '2017-03-20 13:42:13.622', '', NULL

Entry in table ticket_article
id, ticket_id, type_id, sender_id, from, to, cc, subject, message_id, message_id_md5, in_reply_to, content_type, references, body, internal, preferences, updated_by_id, created_by_id, created_at, updated_at
'5417', '1225', '10', '1', 'User', NULL, NULL, NULL, NULL, NULL, NULL, 'text/plain', NULL, 'merged', '0', NULL, '8', '8', '2017-03-20 13:42:13', '2017-03-20 13:42:13.055'

Corresponding SLA table entry
id, name, calendar_id, first_response_time, update_time, solution_time, condition, updated_by_id, created_by_id, created_at, updated_at
'2', 'Rückrufe', '1', '1440', '1440', NULL, '--- !ruby/hash-with-ivars:ActionController::Parameters\nelements:\n  ticket.group_id: !ruby/hash-with-ivars:ActionController::Parameters\n    elements:\n      operator: is\n      value: \'16\'\n    ivars:\n      :@permitted: false\n  ticket.state_id: !ruby/hash-with-ivars:ActionController::Parameters\n    elements:\n      operator: is\n      value: &1\n      - \'1\'\n      - \'2\'\n    ivars:\n      :@permitted: false\n      :@converted_arrays: !ruby/object:Set\n        hash:\n          *1: true\nivars:\n  :@permitted: false\n', '5', '6', '2017-02-16 21:38:45', '2017-02-27 15:40:33.303'

Envolved ticket_states
id, state_type_id, name, next_state_id, ignore_escalation, note, active, updated_by_id, created_by_id, created_at, updated_at, default_create, default_follow_up
'1', '1', 'new', NULL, '0', NULL, '1', '1', '1', '2017-01-02 13:36:37', '2017-02-23 09:25:16.351', '1', '0'
'2', '2', 'open', NULL, '0', NULL, '1', '1', '1', '2017-01-02 13:36:37', '2017-02-23 09:25:16.340', '0', '1'
'5', '6', 'merged', NULL, '1', NULL, '1', '1', '1', '2017-01-02 13:36:37', '2017-02-23 09:25:16.366', '0', '0'

I saved a database dump, if there are more questions.

Greetings
tobibe

**Comment by rolfschmidt** [comment]
Hi @tobibe ,
we had this behaviour sometimes in our support system, but currently we are also not able to reproduce it. If you find a way to reproduce it, please let us now.

You also can use our test systems to play around if you want to https://zammad.com/getting-started

**Comment by tobibe** [comment]
Hello @rolfschmidt ,

I think I found a way.

We configured an SLA to escalate tickets in a specific group and in state new or open.
![zammad-sla](https://cloud.githubusercontent.com/assets/9305993/24611823/a84e5e80-1883-11e7-9d60-f970599ecc15.PNG)

I create a Ticket in this group and state in it shows me that the ticket escalates in the given time.

Now I use a Makro which changes the group to another group, state to open and user to nouser.
![zammad-makro](https://cloud.githubusercontent.com/assets/9305993/24611987/312afde4-1884-11e7-85ae-3635cfb64ae0.PNG)

With the change of the group, the ticket doesn't match the SLA-conditions, but it still shows me the escalation time.

Maybe this could help

Greetings
tobibe
